src/plugins/editor/editor.less

```diff
@@ -6,7 +6,7 @@
   }
 
   &-inner {
-    display: flex;
+    display: grid;
```

Comment:
Probably we need to make sure we change it globally...
The vertical collapsing applicable for all plugins. Settings can be collapsed vertically (and thay are on small breakpoints on ESL)

Reply:
@ala-n, I’ll check it